keps/sig-scheduling/2891-simplified-config/README.md

```diff
@@ -0,0 +1,861 @@
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This KEP defines a simplified field for end-users to configure Scheduler plugins
+which use multiple extension points easily. It is based on a new extension point that
+plugin developers can implement in their plugins. This extension point will automatically
+register a plugin for all (or some) of the scheduling cycle extension points that the
+plugin implements. Thus simplifying the configuration required for cluster administrators
+while preserving the development benefits of the scheduling framework.
+
+
+## Motivation
+
+The [Scheduling Framework](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/624-scheduling-framework) was designed with
+developers of custom schedulers in mind to make the scheduler more extendable. This design
+provided several extension points used by developers to implement their custom plugins into
+the scheduling cycle. Some extension points, like `PreFilter` and `PreScore`, are only
+used for data processing or other internal functions not directly relevant to the end user.
+
+However these "behind-the-scenes" endpoints must still be configured in the `KubeSchedulerConfiguration`
+provided by the user for every plugin that uses them. For the average user this is an unnecessary
+and confusing step -- they only care that the plugin works, not how it works, and so
+internal processing steps are usually irrelevant to them.
+
+Therefore, a way for users to simply toggle all of a plugin's extension points
+(or a subset of the extension points as defined necessary by the plugin developer)
+would remove complexity and the potential for misconfiguration.
+
+### Goals
+
+* Provide an option for simplified plugin config from the end-user's perspective
+* Enable this config in all in-tree default scheduler plugins
+
+### Non-Goals
+
+* Refactor or introduce any breaking changes to the scheduling framework
+
+## Proposal
+
+A new extension point, `MultiPoint` (final name TBD) will be made available to
+users and developers which handles plugin registration for multiple extension
+points simultaneously.
+
+From the developer's perspective, this extension point will be an optional interface to
```

Comment:
Is this the option we are going with?
Even if we are, these details could go as part of the "Design Details" section.